osu.Game/osu.Game.csproj

```diff
@@ -18,7 +18,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.2.1" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="2.2.1" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
-    <PackageReference Include="ppy.osu.Framework" Version="2019.125.0" />
+    <PackageReference Include="ppy.osu.Framework" Version="0.0.8120" />
```

Comment:
You probably did not want to commit this change.

Reply:
That was actually intentional. Not sure how the procedure is but I wanted to keep this functional until the dependent framework PR is merged.